packages/core/faststore.config.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
     workspace: 'master',
     environment: 'vtexcommercestable',
     hideUnavailableItems: true,
-    incrementAddress: false,
+    incrementAddress: true,
```

Comment:
Yes, it would be better since the client would have the same behaviour he already have today! I´ll document this at the documentation I´m doing tomorrow :)

Reference: https://vtex.slack.com/archives/C03L3CRCDC4/p1687963554677969?thread_ts=1687961812.343719&cid=C03L3CRCDC4